src/pybind/mgr/dashboard/controllers/auth.py

```diff
@@ -10,7 +10,8 @@
 
 from .. import mgr
 from ..exceptions import InvalidCredentialsError, UserDoesNotExist
-from ..services.auth import AuthManager, JwtManager
+from ..services.auth.auth import AuthManager, AuthType, BaseAuth, JwtManager
+from ..services.auth.oauth2 import OAuth2
```

Comment:
Same thing here. You could also add `OAuth2` to the `__all__` list, or make it explicit as you did here:
```suggestion
from ..services.auth import AuthManager, AuthType, BaseAuth, JwtManager
from ..services.auth.oauth2 import OAuth2
```

. A good thing of importing through `__init__.py` `__all__` is that you keep the implementation details (e.g.: filenames) hidden:
```suggestion
from ..services.auth import AuthManager, AuthType, BaseAuth, JwtManager, OAuth2
```